charts/kserve-resources/templates/deployment.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
   labels:
     control-plane: kserve-controller-manager
     controller-tools.k8s.io: "1.0"
+    {{- if .Values.kserve.controller.labels }}
```

Comment:
from where this value is coming from?
Maybe we can document it to the users?

Reply:
It will be coming from `values.yaml` (see below), and can be overridden by the users.
I actually followed the same pattern as all the other existing parameter. It seems pretty straightforward to me.
Let me know where I should document it if you think otherwise. Happy to do it.